lib/DBus/Transport.hs

```diff
@@ -77,6 +90,12 @@ class Transport t where
     --
     -- Throws a 'TransportError' if an error occurs.
     transportPut :: t -> ByteString -> IO ()
+    
+    -- | Send a 'ByteString' and Unix file descriptors over the transport.
+    --
+    -- Throws a 'TransportError' if an error occurs.
+    transportPutWithFds :: t -> ByteString -> [Fd] -> IO ()
+    transportPutWithFds t bs _fds = transportPut t bs
```

Comment:
Default implementation for backwards compatibility. The `SocketTransport` instance overrides this with an implementation that actually sends the `Fd`s over the socket.